da.json

```diff
@@ -69,9 +69,9 @@
                 "translate" : "oversæt",
                 "about" : "om fnbr.co",
                 "privacy" : "fortrolighedspolitik",
-                "donate" : "Donere",
-                "discord" : "Discord",
-                "twitter" : "Twitter"
+                "donate" : "donere",
+                "discord" : "discord",
```

Comment:
According to the English string, the first letter of this string must be capitalized. (Brands have to be capitalized anyway.)